tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -151,15 +152,16 @@ <h3>No profile data was found.</h3>
         value: 0,
         observer: '_selectedDatasetChanged'
       },
-      _currentTool: {
-        type: String,
-        computed: '_getCurrentToolName(selectedTool, _activeTools)',
-        notify: true,
-      },
       selectedTool: {
         type: Number,
         notify: true,
         value: 0,
+        observer: '_selectedToolChanged'
+      },
+      currentSelected: {
+        type: Number,  // LSB is index of tools, MSB is index of datasets.
```

Comment:
nit: Could you spell out the acronyms LSB and MSB?

Reply:
done.